px-context-browser.html

```diff
@@ -295,6 +419,11 @@
       }
     },
 
+    _isItemCurrentlyFavorited(item, favorited) {
+      if (!item || !favorited || !Array.isArray(favorited)) return false;
```

Comment:
Can just do `if (!item || !Array.isArray(favorited)) return false;` for first line, !Array.isArray handles the null/undefined case for favorited

Reply:
We have to methods that do the same thing, I think: `_isItemCurrentlyFavorited` and `_isItemFavorited`. Can we refactor to make these one method?

Reply:
👍  :+1: